lib/livebook.ex

```diff
@@ -194,6 +194,14 @@ defmodule Livebook do
     if allowed_uri_schemes = Livebook.Config.allowed_uri_schemes!("LIVEBOOK_ALLOW_URI_SCHEMES") do
       config :livebook, :allowed_uri_schemes, allowed_uri_schemes
     end
+
+    if zti = Livebook.Config.zti!("LIVEBOOK_ZTI") do
```

Comment:
ZTA (for zero trust authentication). :)

Reply:
And I think we will have a single header. Something like:

    LIVEBOOK_ZTA="cloudflare:key"

Reply:
Actually, I think at some point we decided the header should be:

    LIVEBOOK_IDENTITY_PROVIDER="cloudflare:key"

The default value is "cookies".